### DISCUSSION TITLE
Security scanning

### DISCUSSION THREAD

**Discussion by nlebreton** [badges/shields#10073] [Q&A] [answered]
Hi all,

First off all, thank a lot for the shields project! 

Could you tell me if the code is scanned by GitHub Advanced Security or any similar tool?

**Comment by chris48s** [comment, marked as answer]
Short answer: no.

We do use [socket.dev](https://socket.dev/) and https://github.com/actions/dependency-review-action to flag possible security issues in 3rd party packages, but we don't have any analysis in place for our own code.

**Reply by nlebreton** [reply]
Very clear.

Thank you.
